frontend/src/views/Apps/AppDetail/AppDetail.js

```diff
@@ -62,7 +62,7 @@ class AppDetail extends Component {
       networkData,
     } = await ApplicationService.getApplication(address);
 
-    const chains = await NetworkService.getNetworkChains(networkData.chains);
+    const chains = await NetworkService.getAvailableNetworkChains();
```

Comment:
Possible issue here, here is not available networks, here is the networks of the app.